재호/1759.py

```diff
@@ -0,0 +1,32 @@
+# https://www.acmicpc.net/problem/1759
+
+from sys import stdin
+
+
+def check(password):
+    mo, ja = 0, 0
+    for p in password:
+        if p == 'a' or p == 'e' or p == 'i' or p == 'o' or p == 'u':
```

Comment:
```suggestion
        if p in "aeiou":
```
어떤 블로그에서 이런 방식을 사용하는걸 봤어요!! 훨씬 깔금하고 가독성 있는것 같아요~

Reply:
오 진짜 훨씬 더 깔끔하고 좋은 것 같아요! 👍